src/views/Auth/Auth.test.jsx

```diff
@@ -0,0 +1,20 @@
+import { render } from "@testing-library/react"
+import { MemoryRouter, Route, Switch } from "react-router-dom";
+import { UserProvider } from "../../context/UserContext";
+import Auth from "./Auth";
+
+it('should test the code for the login page', async () => {
+const container = render(
```

Comment:
it helps to destructure `{ container }` to avoid superfluous stuff in your snapshots